src/AnalyticsEventReporter.ts

```diff
@@ -66,6 +83,20 @@ export class AnalyticsEventReporter implements AnalyticsEventService {
     const shouldUseBeacon = useBeacon(finalPayload, this.config.forceFetch);
     const requestUrl = setupRequestUrl(this.config.env, this.config.region);
 
+    if (this.config.debug) {
+      console.log(
+        '[DEBUG] The following Payload would be sent to the Yext Events API using ' +
+          (shouldUseBeacon ? 'Beacon. ' : 'fetch(). ') +
+          'Session Tracking is ' +
+          (this.config.sessionTrackingEnabled ? 'enabled.\n' : 'disabled.\n') +
+          'Request URL: ' +
```

Comment:
Is there value in showing the request URL to whoever is using this? Or more value in just showing the config parameters(so they know that they config they set is actually applied).

Reply:
also thoughts on just putting the debug mode logic into the post helper?

Would clean up this method and also make sure that we're not duplicating logic/ the debug could be incorrect(i could see a scenario where we change lines 101 onwards but forget to update the debug log line which is just confusing.